app/assess/templates/macros/fund_dashboard_summary.html

```diff
@@ -53,6 +53,13 @@ <h3 class="govuk-body govuk_summary-card__title govuk-!-font-size-24">{{round_na
             View all {% if is_active_status %}active{% else %}
             closed{% endif %} assessments
         </a>
+        {% if round_application_fields_download_available and access_controller.is_lead_assessor%}
```

Comment:
Itty bitty space missing here:

```suggestion
        {% if round_application_fields_download_available and access_controller.is_lead_assessor %}
```